src/build-using-cmake-and-mxe-on-wsl.tm

```diff
@@ -14,30 +14,25 @@
 
   There are three major steps to build <TeXmacs> on WSL:
 
-  <section*|Step 1. Install Ubuntu 18.04 on Windows 10>
+  <section*|Step 1. Install Ubuntu on Windows 10>
 
-  Ubuntu 18.04 is available in the Windows Store. Just click to install it.
-  For the first time, you will be prompted to provide a username and a
-  password for the installed Ubuntu 18.04 on Windows. The following steps
-  should be executed in the newly-installed Ubuntu 18.04.
+  Ubuntu is available in the Windows Store. Just click to install it. You may
+  first need to install WSL as described <hlink|here|https://docs.microsoft.com/en-us/windows/wsl/install>.
+  When running Ubuntu for the first time, you will be prompted to provide a
+  username and a password for the installed Ubuntu on Windows. The following
+  steps should be executed inside the newly-installed Ubuntu.
 
   <section|Step 2. Install MXE>
 
   <subsection*|Download the source code of MXE to
   <shell|$HOME/github/mxe/mxe>>
 
   <\shell-code>
-    mkdir -p $HOME/github/mxe/mxe
+    mkdir -p $HOME/github/mxe/
 
-    cd $HOME/github/mxe/mxe
+    cd $HOME/github/mxe/
 
-    git init
-
-    git remote add origin https://github.com/mxe/mxe.git
-
-    git fetch origin
-
-    git checkout master
+    git clone https://github.com/mxe/mxe.git
```

Comment:
For example, to clone the android git repo, if you use git clone, you might fail because of the network issue, using git fetch, just fetch again and again, you will finally success. But if you use git clone, you might never success.

Reply:
Not a big issue for most of the Github users. It is just a taste of mine based on the unstable network I'm suffering.

I think the final solution is that creating a Git host server in a decentralized way just like Mastodon.

Reply:
Federated git forges would be awesome! Forgefed is going in that direction. Forgejo seems to be implementing it. Yet another reason to move to Codeberg 🙂

Reply:
Forgejo sounds like an Esperanto name.

Reply:
Thanks, that's exactly what I need.